pkg/cmd/cli/secrets/new.go

```diff
@@ -39,7 +39,10 @@ using with all valid keys in that directory.
   $ %[1]s my-secret path/to/bar
 
   // Create a new .dockercfg secret named my-secret
-  $ %[1]s my-secret path/to/.dockercfg`
+  $ %[1]s my-secret path/to/.dockercfg
+
+  // Create a new .docker/config.json secret named my-secret
```

Comment:
New standard uses `#`.  You can make a separate issue to sweep them all if you like.